keel-plugin/src/main/kotlin/com/netflix/spinnaker/keel/plugin/ResourceHandler.kt

```diff
@@ -164,10 +164,10 @@ abstract class ResourceHandler<S : ResourceSpec, R : Any>(
   }
 
   /**
-   * @return `true` if this plugin is still busy running a previous actuation for the resource
-   * associated with [id], `false` otherwise.
+   * @return `true` if this plugin is still busy running a previous actuation for [resource],
+   * `false` otherwise.
    */
-  open suspend fun <T : ResourceSpec> actuationInProgress(resource: Resource<T>): Boolean = false
+  open suspend fun actuationInProgress(resource: Resource<S>): Boolean = false
```

Comment:
Looks like this method was previously changed from a resource id parameter to the full resource, but the generic type was not kept consistent with the handler's existing generic type.